sys/posix/Kconfig

```diff
@@ -12,5 +12,6 @@ config MODULE_POSIX_HEADERS
     bool
 
 rsource "sleep/Kconfig"
+rsource "inet/Kconfig"
```

Comment:
Can we keep alphabetical order?

Reply:
Maybe we can add in a script... that will be the next thing I do.